pkg/platform/aws/aws.go

```diff
@@ -171,6 +177,12 @@ func createTerraformConfigFile(cfg *config, terraformRootDir string) error {
 		return errors.Wrapf(err, "failed to marshal CLC snippets")
 	}
 
+	// configure oidc flags and set it to KubeAPIServerExtraFlags.
+	if cfg.Oidc != nil {
+		oidcFlags, _ := cfg.Oidc.ToKubeAPIServerFlags(cfg.clusterDomain())
```

Comment:
Maybe we could mention here in the comment, that we can skip validation, because it's done in `checkValidConfig()` ?